### ISSUE TITLE
MinesweeperCell.open has no test

### ISSUE THREAD

**Issue by lbuttignol** [nmaguirre/minesweeper#117]
MinesweeperCell.open should have tests checking its expected functionality in a number of different scenarios. Statement coverage is expected from the tests of this routine.

**Comment by lbuttignol** [comment]
Solved in this commit  cd6e539

**Comment by nricci** [comment]
@lbuttignol: I'm not sure about `openACellWithMines` expecting an exception. As I imagine it, it is perfectly fine to open a cell with a mine. It is a special case since that is the moment in the game when the player looses, but is a valid one nonetheless

**Comment by lbuttignol** [comment]
Yes, you're right. I thought that having a mine should throw an exception to terminate the program , but in this case it would be broken. In this case only must check that the cell must have a mine, and delete the exception expected.

**Comment by lbuttignol** [comment]
Solved in this rev. 0a0d936
